uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-container.js

```diff
@@ -1,22 +1,28 @@
-define([
-  'leave-absences/absence-tab/modules/components'
-], function (components) {
-  components.component('absenceTabContainer', {
-    bindings: {
-      contactId: '<'
-    },
-    templateUrl: ['settings', function (settings) {
-      return settings.pathTpl + 'components/absence-tab-container.html';
-    }],
-    controllerAs: 'absence',
-    controller: ['$log', '$rootScope', function ($log, $rootScope) {
-      $log.debug('Component: absence-tab-container');
+/* eslint-env amd */
 
-      $rootScope.role = 'admin';
+(function (CRM) {
+  define([
+    'leave-absences/absence-tab/modules/components'
+  ], function (components) {
+    components.component('absenceTabContainer', {
+      bindings: {
+        contactId: '<'
```

Comment:
No we don't. We use `$root.settings.contactId` in templates. I removed it and ran tests - all good.